hack/generate-cr.sh

```diff
@@ -68,24 +81,32 @@ spec:
       redirectURI: "https://${NAME}-${NS}.${APPS}/callback"
 EOF
 
-cat > demo-dexclient-hub.yaml <<EOF
```

Comment:
The Readme still references this file. Could you update it please?

Reply:
Updated